packages/teleport/src/services/nodes/makeNode.ts

```diff
@@ -14,25 +14,20 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-import { at } from 'lodash';
 import { Node } from './types';
 
 export default function makeNode(json): Node {
-  const [id, clusterId, hostname, tags = [], addr, tunnel = false] = at(json, [
-    'id',
-    'siteId',
-    'hostname',
-    'tags',
-    'addr',
-    'tunnel',
-  ]);
+  const {id, clusterId, hostname, addr, tunnel} = json;
+
+  const tags = json.tags || [];
+  const tagTexts = json.tagTexts || tags.map(tag => `${tag.name}: ${tag.value}`)
```

Comment:
the json response won't contain a field named `tagTexts`, so checking if that is undefined is not needed, and since we supplied a default value for field `tags` (if `tags` came undefined), we can just do `tags.map` without worrying about undefined/null errors

Reply:
Yea, I wrote this before I found out the tests don't go through `makeNode`, so I though the `tagTexts` could be passed in